src/chainparams.cpp

```diff
@@ -255,54 +255,18 @@ class CRegTestParams : public CChainParams {
 public:
     explicit CRegTestParams(const ArgsManager& args) {
         strNetworkID = "regtest";
```

Comment:
nit: could set this string from `CBaseChainParams::REGTEST`.

Reply:
mhmm. I remember this failing to compile or to run at some point, but it seems to work now. Perhaps it had to do with some older version of cpp or something. Or perhaps I misremember and it was only networksyle.cpp where I wasn't able to to it.
In any case, since this can be done for other chains other than regtest and in other places other than chainparams.cpp, I'm going to do it in another PR: https://github.com/bitcoin/bitcoin/pull/17306